### ISSUE TITLE
New quickstart generates wrong SPHINXBUILD in Makefile

### ISSUE THREAD

**Issue by tk0miya** [sphinx-doc/sphinx#3965]
### Problem
- 

#### Procedure to reproduce the problem
```
$ docker run -it --rm ubuntu:trusty bash
# apt update
# apt install python3-pip
# pip3 install sphinx
# sphinx-quickstart /doc
(snip)
# cd /doc
# make html
/bin/sh: 1: python: not found
make: *** [html] Error 127
```

#### Error logs / results
Same above.

#### Expected results
- Build suceeds.

### Environment info
- OS: Ubuntu trusty
- Python version: 3.4
- Sphinx version: 1.6.3

**Comment by tk0miya** [comment]
Note: this comes from #3523

**Comment by peterjc** [comment]
This also breaks if sphinx is installed under Python 3 but not under Python 2 (i.e. if ``python`` defaults to the wrong Python)

**Comment by tk0miya** [comment]
Fixed at a5f6e97ac45726dcad9dd267279b04d9539e7793.
